tests/unit/util/test_file_converter.py

```diff
@@ -134,13 +161,35 @@ def test_yaml_remove_nan_names(saved_yaml):
     output (including keys)."""
     # Note: This test works specifically with this test file as the second
     # set of names in the file are nans.
-    keys_not_required = ['firstname2', 'surname2']
-    for key in keys_not_required:
-        assert key not in saved_yaml['authors']
+    assert len(saved_yaml['authors']) == 1
 
 
 def test_yaml_datetime_format(saved_yaml):
     """Test that saved yaml file contains dates in isoformat."""
     assert dt.datetime.fromisoformat(saved_yaml["time range"]["start"])
     assert dt.datetime.fromisoformat(saved_yaml["time range"]["end"])
 
+
+def test_bad_output_type(excel_filepath, tmp_output_path):
+    """Test that an exception is raised when an invalid filetype is
+    specified (or not specified at all)."""
+    with pytest.raises(Exception):
+        fc.convert_excel(excel_filepath, tmp_output_path, 'none')
+
+
+def test_csv_no_index(tmp_output_path, netcdf_filepath):
+    """Test that indexes have not been added to the converted csv file."""
+    # NOTE: Due to the behaviour of the csv reader, this file can only be
+    # examined within the condition 'with open...', so more set-up tasks must
+    # be performed here as opposed to inside a fixture.
+    csv_fname = tmp_output_path / "flightpath.csv"
+    fc.convert_netcdf(netcdf_filepath, csv_fname)
+    with open(csv_fname, newline='') as csvfile:
+        saved_csv = csv.reader(csvfile, delimiter=' ')
```

Comment:
I don't think this is right. The file came from `Dataframe.to_csv` so the delimiter should be comma, not space (or just don't specify it and `csv.reader` should guess correctly.

I think this is actually splitting the datetime column in half, because pandas probably saved it as `yyyy-mm-dd HH:MM:SS`. Which I don't think strictly speaking is isoformat (separator between date and time must be "T") but in practice is common anyway as it's more readable. So the test would probably fail if changed (and only passes currently because the time part is entirely optional)

Might have to use `datetime.strptime` with the expected format. Or at least explain why it's being split on space not comma.